torch/csrc/distributed/rpc/request_callback_no_python.cpp

```diff
@@ -517,7 +517,7 @@ void RequestCallbackNoPython::processRpc(
                 }
                 if (cuda_profiling_enabled &&
                     0 == strcmp(e.name(), "__cuda_start_event")) {
-                  e.setCudaUs(e.cpu_us());
+                  e.setCudaUs(e.cpuUs());
```

Comment:
just fyi, this might have some merge conflict with https://github.com/pytorch/pytorch/pull/44664 (which is just moving this function to a different file, so should be an easy fix)